_component_design/lists.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Lists
+parent: Content Areas
+layout: component
+category: Components
+usage: >
+  Lists help break up large blocks of text, make complex articles and blog posts easier to grasp, and make key information stand out.
+preview-image: preview-images/list.svg
+resource: true
+status: stable
+last-modified: 2017-11-16
+---
+
+<section class="static-section"  markdown="1">
+
+## Lists
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Unordered lists
+
+Use unordered lists to group related items Items in a list should share a similar level of importance. When possible, write bullet items to be approximately the same length. Keeping a uniform length is pleasing to the eye and makes the list more consumable to a user.
```

Comment:
Period missing. s/`to group related items Items in a list`/`to group related items. Items in a list`

Reply:
updated